lib/CryptoLD.js

```diff
@@ -127,6 +148,28 @@ class CryptoLD {
   _installed({type}) {
     return this.suites.has(type);
   }
+
+  /**
+   * Returns the installed crypto suite class for a given document's type.
+   *
+   * @param {object} document - A serialized key document (or options document).
```

Comment:
I think we're playing a little fast and loose with this API (maybe you pass a doc, maybe you pass some options, if the thing you pass isn't defined its treated the same as no type being passed ... and so on). I could see someone getting tripped up on this in the future. So we might want it to be a little cleaner, but won't hold up the PR for it.

Reply:
Created issue.